### PR TITLE
US9 nav bar 

### PR DESCRIPTION
**Pull request by gwittm**

### CODE REVIEW

File: components/Search/FuseSearchRecipe.js [hunk 1]
```diff
@@ -1,15 +1,21 @@
 import AllRecipesList from "../AllRecipesList";
 import { useState, useEffect } from "react";
```
Comment (Philipp-Hess) [inline components/Search/FuseSearchRecipe.js:2]: maybe u should remove the useEffect import here

File: components/NavBar/index.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+export default function NavBar() {
+  const [isHomeActive, setIsHomeActive] = useState(true);
+  const [isNewRecipeActive, setIsNewRecipeActive] = useState(false);
+
+  const handleToggle = (page) => {
+    if (page === "home") {
+      setIsHomeActive(true);
+      setIsNewRecipeActive(false);
+    } else {
+      page === "newRecipe";
```
Comment (Philipp-Hess) [inline components/NavBar/index.js:13]: maybe u could remove this line

**Review by Philipp-Hess — COMMENTED** [review]
Very nice code 💯 -  i only have minor suggestions

File: components/Search/FuseSearchRecipe.js (hunk 1 above)
Comment (gwittm) [reply components/Search/FuseSearchRecipe.js:2]: useEffect is used in this file, so we need it ;)

**Comment by Philipp-Hess** [comment]
LGTM 🙌

**Pull request merged**